js/freewheel.js

```diff
@@ -304,7 +305,7 @@ OO.Ads.manager(function(_, $) {
     var _setAdRequestTimeout = _.bind(function(callback, duration){
       if (adRequestTimeout) {
         var error = "Ad Request Timeout already exists - bad state";
-        fw_onError(error);
+        fw_onError(null, error);
```

Comment:
So the fw_onError function on line 816 takes in an event object and a string error message. The function doesn't actually do anything with the event, so I passed in null. Should I change the original function?